cylc/flow/task_job_mgr.py

```diff
@@ -394,7 +394,7 @@ def submit_task_jobs(self, suite, itasks, curve_auth,
                     # write flag so that subsequent manual retrigger will
                     # generate a new job file.
                     itask.local_job_file_path = None
-                    itask.state.reset(TASK_STATUS_PREPARING)
```

Comment:
"queue-released" tasks remained in the queued state until this point, after remote init and remote file install. Now setting to "preparing" immediately on queue release.